iroh/examples/client.rs

```diff
@@ -24,7 +24,8 @@ async fn main() -> anyhow::Result<()> {
     let mut stream = doc.get_many(Query::all()).await?;
     while let Some(entry) = stream.try_next().await? {
         println!("entry {}", fmt_entry(&entry));
-        let content = entry.content_bytes(client).await?;
+        // You can pass either a `Doc` or the `Iroh client`.
```

Comment:
```suggestion
        // You can pass either `&doc` or the `client`.
```
This uses the actual names used here, which is probably better.